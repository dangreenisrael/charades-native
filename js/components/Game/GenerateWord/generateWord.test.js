import 'react-native';
import renderer from 'react-test-renderer';
import React from 'react';
import {GenerateWord} from '.';

describe('Generate Word', () => {
  it('renders correctly', () => {
    const mock = jest.fn();
    const tree = renderer
      .create(<GenerateWord category="Things" updateDictionary={mock} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
