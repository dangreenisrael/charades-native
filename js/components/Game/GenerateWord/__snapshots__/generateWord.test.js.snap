// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Generate Word renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  isTVSelectable={true}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "center",
      "backgroundColor": "#007aff",
      "borderBottomWidth": null,
      "borderColor": "#007aff",
      "borderLeftWidth": null,
      "borderRadius": 5,
      "borderRightWidth": null,
      "borderTopWidth": null,
      "elevation": 2,
      "flexDirection": "row",
      "height": 60,
      "justifyContent": "space-between",
      "opacity": 1,
      "paddingBottom": 6,
      "paddingLeft": 16,
      "paddingRight": 16,
      "paddingTop": 6,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#fff",
        "fontFamily": "System",
        "fontSize": 22,
        "lineHeight": 32,
        "marginLeft": 0,
        "marginRight": 0,
      }
    }
    virtual={undefined}
  >
    Generate 
    Thing
  </Text>
</View>
`;
